clusters/resource_cluster.go

```diff
@@ -244,6 +293,11 @@ func (ClusterSpec) CustomizeSchemaResourceSpecific(s *common.CustomizableSchema)
 		Type:     schema.TypeString,
 		Computed: true,
 	})
+	s.AddNewField("autotermination_minutes", &schema.Schema{
+		Type:     schema.TypeInt,
+		Optional: true,
+		Default:  60,
+	})
```

Comment:
Only adding it as a resource specific field for cluster because job resource never needs it